src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5044,4 +5044,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_CompilerAndLanguageVersion" xml:space="preserve">
     <value>Compiler version: '{0}'. Language version: {1}.</value>
   </data>
+  <data name="WRN_Experimental" xml:space="preserve">
```

Comment:
I'm surprised some test didn't force you to add a corresponding _Title entry too.